### PR TITLE
Add intl check in bootstrap.

### PR DESCRIPTION
**Pull request by markstory**
This check should help people understand what they need to do better than 'Missing class Locale' does. Refs cakephp/cakephp#5553

### CODE REVIEW

File: config/bootstrap.php [hunk 1]
```diff
@@ -33,6 +33,11 @@
  */
 require CORE_PATH . 'config' . DS . 'bootstrap.php';
 
+// You can remove this if you have confident you have intl installed.
```
Comment (dereuromark) [inline config/bootstrap.php]: "have confidence" or "are confident" :)

**Comment by dereuromark** [comment]
:+1:

**Comment by antograssiot** [comment]
don't this must be before the require autoload.php ?

File: config/bootstrap.php (hunk 1 above)
Comment (markstory) [inline config/bootstrap.php]: Derp...

**Comment by markstory** [comment]
@antograssiot why? Before the autoloader? The locale class won't be called until it is first used. I will uninstall my intl ext to see if I am dumb.

**Comment by antograssiot** [comment]
@markstory sorry I'm the dumb one, tested it to fast. didn't mean wasted your time

**Pull request merged**